h2o-genmodel/src/main/java/hex/genmodel/ModelMojoReader.java

```diff
@@ -251,6 +252,7 @@ else if (line.equals("[columns]")) {
     return info;
   }
 
+  private Pattern CRLF_ONLY_LINE = Pattern.compile("[\r\n]*");
```

Comment:
this will also match an empty string, if that is expected it should be part of the test

Reply:
Yes, this is intended and it will for sure be part of the test. Thanks for noticing. Once we know this is the way to go, I'll finish the PR. Sorry for not making it obvious.